src/test/java/com/google/api/codegen/gapic/testdata/py/python_multiple_services.baseline

```diff
@@ -1555,27 +1555,11 @@ import sys
 
 from google.api_core.protobuf_helpers import get_messages
 
-from google.api import http_pb2
```

Comment:
Are these changes safe?

Reply:
Ok, looking at google-cloud-python, I agree that longrunning, metadata, resources, signature, and descriptor *pb2 imports are unused. I'm not sure about the the API proto imports.